be/src/olap/olap_cond.h

```diff
@@ -87,11 +88,13 @@ struct Cond {
     }
 
     CondOp op;
-    // valid when op is not OP_IN
+    // valid when op is not OP_IN and OP_NOT_IN
     WrapperField* operand_field;
-    // valid when op is OP_IN
+    // valid when op is OP_IN or OP_NOT_IN
     typedef std::unordered_set<const WrapperField*, FieldHash, FieldEqual> FieldSet;
     FieldSet operand_set;
+    WrapperField* min_value_field;
```

Comment:
Add comment to explains these 2 new fields.

Reply:
done